src/argilla/client/feedback/schemas/metadata.py

```diff
@@ -26,6 +26,7 @@
     root_validator,
     validator,
 )
+from pydantic.generics import GenericModel
```

Comment:
Unused import

```suggestion
```

Reply:
Mmmmm. The pre-commit step should remove them

Reply:
Right now `ruff` is not configured to remove unused imports, once we merge the metadata feature branch we could try to merge this one https://github.com/argilla-io/argilla/pull/3478 that adds some more linting rules, including the one of the unused imports